core/src/main/java/jeeves/config/springutil/JeevesNodeAwareLogoutSuccessHandler.java

```diff
@@ -84,8 +85,12 @@ protected String determineTargetUrl(HttpServletRequest request, HttpServletRespo
 
                     String siteHost = settingManager.getValue(Settings.SYSTEM_SERVER_HOST);
                     String siteProtocol = settingManager.getValue(Settings.SYSTEM_SERVER_PROTOCOL);
-                    int sitePort = settingManager.getValueAsInt(Settings.SYSTEM_SERVER_PORT);
-
+                    
+                    // some conditional logic to handle the case where there's no port in the settings
+                    int sitePort = 80;
+                    if (Settings.SYSTEM_SERVER_PORT != null && StringUtils.isNumeric(Settings.SYSTEM_SERVER_PORT)) {
```

Comment:
```suggestion
                    if (StringUtils.isNumeric(Settings.SYSTEM_SERVER_PORT)) {
```

`StringUtils.isNumeric` already checks `null` value.